geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/package.scala

```diff
@@ -24,7 +24,7 @@ import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 package object geotools {
 
   // use the epsg jar if it's available (e.g. in geoserver), otherwise use the less-rich constant
-  val CRS_EPSG_4326 = try { CRS.decode("EPSG:4326") } catch { case t: Throwable => DefaultGeographicCRS.WGS84 }
+  val CRS_EPSG_4326 = try { CRS.decode("EPSG:4326", true) } catch { case t: Throwable => DefaultGeographicCRS.WGS84 }
```

Comment:
I guess the test changed b/c of this change - is this going to cause any other change in behavior? We should probably document it in the upgrade guide

Reply:
Ok, so this is *subtle*.  This line over here.... https://github.com/locationtech/geomesa/blob/master/geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/geotools/MetadataBackedDataStore.scala#L43  can change the way that this executes.

In situations where we have a 'big-data datastore', the other line is likely to be called before memoize the variable CRS_EPSG_4326.  That means, that we get a 4326 which is X/Y.  Otherwise, by default, one gets Y/X.

Reply:
sounds good to me